src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/SchemaDef.cs

```diff
@@ -674,7 +674,7 @@ class SchemaDef
             name: "TypeInstantiationSignature",
             members: new MemberDef[] {
                 new MemberDef("GenericType", TypeDefOrRefOrSpec, MemberDefFlags.RecordRef),
-                new MemberDef("GenericTypeArguments", TypeDefOrRefOrSpec, MemberDefFlags.List | MemberDefFlags.RecordRef),
+                new MemberDef("GenericTypeArguments", TypeDefOrRefOrSpecOrMod, MemberDefFlags.List | MemberDefFlags.RecordRef),
```

Comment:
This was forced because an IL test was asserting the compiler. This is invalid per ECMA-335.

Reply:
This should be covered in https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md#3-custom-modifiers-can-go-in-more-places-than-specified

Make the same change for ArraySignature below?